packages/react-examples/src/react/DetailsList/docs/DetailsListBestPractices.md

```diff
@@ -13,6 +13,47 @@
 
 In addition to creating column headers, DetailsList also allows the manual definition of row headers. In the example below, the Name column has been specified as a row header using `isRowHeader: true`. When creating a DetailsList where one column is clearly the primary label for the row, it's best to use `isRowHeader` on that column to create a better screen reader experience navigating the table. For selectable DetailsLists, specifying a row header also gives the checkboxes a better accessible label.
 
+### Keyboard / Hotkeys
+
+DetailsList supports different selection modes with keyboard behavior differing based on the current selection mode.
+
+**Note**: On macOS systems the `ctrl` key should be replaced with `cmd`.
```

Comment:
I actually think this can be left off -- even though `cmd` works on mac, since `cmd + space` is spotlight search, `ctrl` is actually more reliable for detailslist selection